packages/verifiable-credentials/src/sharing-credential.ts

```diff
@@ -24,10 +25,11 @@ export default class SharingCredential {
 	/**
 	 * Method to encrypt and issue credential
 	 * @param didJwtVc
+	 * @param options
 	 * @returns
 	 */
 
-	async issueEncryptedPresentation(didJwtVc: any): Promise<any> {
+	async issueEncryptedPresentation(didJwtVc: any, options: CreatePresentationOptions = {removeOriginalFields: false}): Promise<any> {
```

Comment:
No need to set the default value for `options` as it will eventually use the defauls from `this.issuePublicPresentation`

Reply:
Done

Reply:
You removed the default value on the wrong method

Reply:
Oh, yeah. Done